tests/test_iclr_conference_v2.py

```diff
@@ -270,6 +270,18 @@ def test_post_submission(self, client, openreview_client, helpers):
         assert submissions[0].readers == ['everyone']
         assert '_bibtex' in submissions[0].content
         assert 'author={Anonymous}' in submissions[0].content['_bibtex']['value']
+        
+        # Assert that activation date of paper matching setup inv == abstract deadline
+        request_form=pc_client.get_notes(invitation='openreview.net/Support/-/Request_Form')[0]
```

Comment:
you have the value of `abstract_date` in your test, you can use it to assert the matching_invitation.cdate. 

Otherwise you are running the same code you that are trying to test.

Reply:
Ah I see now, it's redundant. I'll fix it.

Reply:
You already have an instance of request_form in line 255: https://github.com/openreview/openreview-py/pull/2000/files#diff-28496bc067ac20c984a751f05b5894ae94a1323eb2aa4b23b7c12a60bda1e2a1L225

Reply:
@melisabok Yes but the request form is updated right after. I wanted to test the activation date after the abstract deadline had been set to the past, but I can move it?